runhouse/servers/cluster_servlet.py

```diff
@@ -312,6 +312,21 @@ async def aclear_all_references_to_servlet_name(self, servlet_name: str):
             ]
             for key in deleted_keys:
                 self._key_to_servlet_name.pop(key)
+
+            # remove from servlet cache in all processes
+            await asyncio.gather(
+                *[
+                    obj_store.acall_servlet_method(
+                        servlet,
+                        "aremove_servlet_from_cache",
+                        servlet_name,
+                    )
+                    for servlet in self._initialized_servlet_args
```

Comment:
dumb nit, but we have a wrapper for that
```suggestion
                    for servlet in self.aget_all_initialized_servlet_args()
```